src/OpenTelemetry/Metrics/Exemplar/ExemplarFilters.cs

```diff
@@ -0,0 +1,39 @@
+// <copyright file="ExemplarFilters.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Metrics;
+
+/// <summary>
+/// Provides access to built-in ExemplarFilters.
+/// </summary>
+internal static class ExemplarFilters
```

Comment:
```suggestion
internal static class BuiltinExemplarFilters
```

Reply:
removed the extra class and made it part of ExemplarFilter itself.